nautobot/docs/release-notes/version-1.0.md

```diff
@@ -76,14 +76,25 @@ For example, a VLAN is mapped to a Site by default.  After a VLAN is created tod
 
 All of the individual Django apps in NetBox (`dcim`, `extras`, `ipam`, etc.) have been moved into a common `nautobot` Python package namespace. The `netbox` application namespace has been moved to `nautobot.core`. This will require updates when porting NetBox custom scripts and reports to Nautobot jobs, as well as when porting NetBox plugins to Nautobot.
 
+#### Packaging Changes
```

Comment:
Nit: I'd originally ordered all of the subheadings alphabetically, to bypass my own uncertainty about "importance order" or anything like that. I'm not opposed to moving these two to the top, just FYI.

Reply:
Ahh yeah I was just thinking logical flow order and not alpha sort. :/